prow/config.yaml

```diff
@@ -1260,6 +1260,32 @@ postsubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  - name: maintenance-boskos-config-upload
+    agent: kubernetes
+    branches:
+    - master
```

Comment:
Does this make this post submit job run only when changes are merged to the master branch or does it do something else?